jsonService/src/main/webapp/passfault.css

```diff
@@ -215,7 +215,6 @@ body {
 	clear: both;
 	padding: 1.5em;
 	max-width: 48em;
-	height: 100em;
```

Comment:
I had to try it out to remember why I did this.  It's a hack really.  I'd like a better solution.  However, without this, when I hover over a (white) pattern sign with the mouse, it shows another (blue) sign below - off the screen.  When scrolling down to see the (blue) sign it disappears before you can read it.

The "hover over" is really a bad experience anyway.  It would be better to put an "info" icon in the corner of the sign, so clicking or tapping it would put a pop-up with the pattern information.